dts/bindings/serial/atmel,sam0-uart.yaml

```diff
@@ -22,12 +22,16 @@ properties:
   rxpo:
     type: int
     required: true
-    description: Receive Data Pinout
+    description: Receive Data Pinout. The SERCOM_PAD number used for RX.
```

Comment:
Let's create the table from datasheet in the description. This will be very helpful indeed.

![image](https://user-images.githubusercontent.com/7027887/220405244-878be2ed-adc9-4f2a-ae1e-e20fca0003ce.png)